ComponentKit/DataSources/CKCollectionViewTransactionalDataSource.mm

```diff
@@ -159,8 +202,7 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
 {
   CKCollectionViewDataSourceCell *cell = [_collectionView dequeueReusableCellWithReuseIdentifier:kReuseIdentifier forIndexPath:indexPath];
   CKTransactionalComponentDataSourceItem *item = [_currentState objectAtIndexPath:indexPath];
-  attachToCell(cell, item, _attachController);
-  [_cellToItemMap setObject:item forKey:cell];
```

Comment:
Oops! Good catch. I think this might've been an error during rebasing. It turns out that `_cellToItemMap` was being neglected in other circumstances as well, so I will go ahead and fix that as well.